src/mrpro/utils/filters.py

```diff
@@ -1,6 +1,6 @@
 """Spatial and temporal filters."""
 
-# Copyright 2023 Physikalisch-Technische Bundesanstalt
+# Copyright 2024 Physikalisch-Technische Bundesanstalt
```

Comment:
afaik this should be kept as 2023, as the file was initially written 2023 and we want to claim it from 2023 onwards...